Sources/Tasks/Utf8EntityDecode.php

```diff
@@ -0,0 +1,398 @@
+<?php
+
+/**
+ * Simple Machines Forum (SMF)
+ *
+ * @package SMF
+ * @author Simple Machines https://www.simplemachines.org
+ * @copyright 2025 Simple Machines and individual contributors
+ * @license https://www.simplemachines.org/about/smf/license.php BSD
+ *
+ * @version 3.0 Alpha 3
+ */
+
+declare(strict_types=1);
+
+namespace SMF\Tasks;
+
+use SMF\Db\DatabaseApi as Db;
+use SMF\Sapi;
+use SMF\Taskrunner;
+
+/**
+ *
+ */
+class Utf8EntityDecode extends BackgroundTask
+{
+	/*****************
+	 * Class constants
+	 *****************/
+
+	public const LIMIT = 500;
+
+	/****************
+	 * Public methods
+	 ****************/
+
+	/**
+	 * This executes the task.
+	 *
+	 * @return bool Always returns true.
+	 * @todo PHP 8.2: This can be changed to return type: true.
+	 */
+	public function execute(): bool
+	{
+		// Avoid leaving data in an inconsistent state.
+		ignore_user_abort(true);
```

Comment:
Minor detail, another PR maybe.  We should have a handler that goes into Sapi do this since we are instructing the Server API to ignore if the request aborts.

Reply:
That makes sense to me, sure.

Reply:
On further consideration, I'm not sure that would be necessary after all. It's such a simple function. Why would we need to wrap it?

Reply:
We give instructions to the Sapi more or less consistently and ensure we perform them in class. But it's a minor thing; we could ignore it for now and handle it later.